testsuite/tests/kuadrant/authorino/operator/tls/conftest.py

```diff
@@ -8,12 +9,29 @@
 
 
 @pytest.fixture(scope="session")
-def certificates(cfssl, authorino_domain, wildcard_domain):
+def cert_attributes() -> Dict[str, str]:
+    """Certificate attributes"""
+    return dict(O="Organization Test",
+                OU="Unit Test",
+                L="Location Test",
+                ST="State Test",
+                C="Country Test")
+
+
+@pytest.fixture(scope="session")
+def cert_attributes_custom(cert_attributes) -> Dict[str, str]:
```

Comment:
It looks like this method should be called `cert_attributes_other`, as you cannot really customise the values. The values will always be whatever you supplied in the provided cert, suffixed by `-other`.

```suggestion
def cert_attributes_other(cert_attributes) -> Dict[str, str]:
```

Reply:
You can actually customize them, in `pytest` you can override fixtures in the individual test files so it is expected that some other test case will change this fixture and supply different attributes. 

From a naming perspective tho, I have no preference either way